apps/src/util/letterCompare.js

```diff
@@ -0,0 +1,28 @@
+const collator = new Intl.Collator();
```

Comment:
I appreciate these being moved into a helper file! This file location makes me think we will have other uses for it? And if that's the case, will we need any other comparators, and should the file be named something more generic? None of these Q's are blocking or require a refactor, just would like to hear your thoughts.

Reply:
This function could be useful anytime someone wants to sort words in this way, putting actual letters first. I'm not necessarily expecting additional comparator functions to be added (YAGNI and all). Should that change, the file could be renamed or the new comparator could be put in a separate file, whichever seems like the best idea for the project at the time.